sso-samples/oidc-sso-sample/pickup-manager/src/main/resources/manager.properties

```diff
@@ -2,15 +2,15 @@ consumerKey=c3dpZnRhcHA=
 consumerSecret=ZGlzcGF0Y2gxMjM0
 
 callBackUrl=http://localhost.com:8080/pickup-manager/oauth2client
-scope=openid internal_application_mgt_view
+scope=openid email profile internal_application_mgt_view
 authzGrantType=code
 
 enableOIDCSessionManagement=false
 enableOIDCBackchannelLogout=true
-authzEndpoint=https://localhost:9443/oauth2/authorize
-OIDC_LOGOUT_ENDPOINT=https://localhost:9443/oidc/logout
-sessionIFrameEndpoint=https://localhost:9443/oidc/checksession
-tokenEndpoint=https://localhost:9443/oauth2/token
+authzEndpoint=https://localhost.com:9443/oauth2/authorize
```

Comment:
Are we instructing to add the `locahost.com` to the environment variables?

Reply:
Yes this has to be added to the /etc/hosts file. Instructions are given.